django-verdant/rca/templates/rca/includes/enquiry_form.html

```diff
@@ -0,0 +1,48 @@
+{% load wagtailcore_tags widget_tweaks %}
+
+{% if enquiry_page %}
+  <form action="{% pageurl enquiry_page %}" method="POST">
+  {% csrf_token %}
```

Comment:
@balazs-endresz unfortunately we can't disable csrf token validation for `FromPage`, because we can't apply the `csrf_exempt` decorator on the `serve` method. See https://github.com/torchbox/wagtail/issues/3066 for details.

So we will request a form page using the AJAX (see changes in `django-verdant/rca_ee/models.py`).